hs_core/models.py

```diff
@@ -103,6 +103,8 @@ def can_view(self, request):
         # have to do import locally to avoid circular import
         from hs_core.views.utils import authorize, ACTION_TO_AUTHORIZE
         return authorize(request, self.short_id, needed_permission=ACTION_TO_AUTHORIZE.VIEW_RESOURCE,
+                         raises_exception=False)[1] or \
+               authorize(request, self.short_id, needed_permission=ACTION_TO_AUTHORIZE.VIEW_METADATA,
```

Comment:
@pkdash I think authorize() with needed_permission being ACTION_TO_AUTHORIZE.VIEW_METADATA alone is sufficient here, so "or" is not needed.